components/dependency_injection.rst

```diff
@@ -170,6 +170,30 @@ like this::
 
     $newsletterManager = $containerBuilder->get('newsletter_manager');
 
+Behavior When Service Does Not Exist
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, an exception is throw when a service does not exist. You can override the behavior by specifying it::
+
+    use Symfony\Component\DependencyInjection\ContainerBuilder;
+    use Symfony\Component\DependencyInjection\ContainerInterface;
+
+    $containerBuilder = new ContainerBuilder();
+
+    // ...
+
+    // the second argument is optional and define behavior
+    $newsletterManager = $containerBuilder->get('newsletter_manager', ContainerInterface::EXCEPTION_ON_INVALID_REFERENCE);
+
+
+Possible behaviors are:
+
+ * ``ContainerInterface::RUNTIME_EXCEPTION_ON_INVALID_REFERENCE``: Throws runtime exception
+ * ``ContainerInterface::EXCEPTION_ON_INVALID_REFERENCE``: Throws an exception **default**
```

Comment:
I don't understand differences between `RUNTIME_EXCEPTION_ON_INVALID_REFERENCE` and `EXCEPTION_ON_INVALID_REFERENCE` in code source.
It's useful for Controllers but not only I guess

Original PR https://github.com/symfony/symfony/pull/26627

Reply:
ping @nicolas-grekas

Reply:
EXCEPTION_ON_INVALID_REFERENCE throws the exception at compile time
RUNTIME_EXCEPTION_ON_INVALID_REFERENCE throws it at runtime - I would document it last